### PR TITLE
Another take at 0.2.2

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by kiwi0fruit** [comment]
Looks like it works but the patch redundantly patches itself. If to remove it would be better.

**Comment by kiwi0fruit** [comment]
And part of the patch that patches download url for windows and revision should be removed.

**Comment by kiwi0fruit** [comment]
Oh. You have already removed it. What's left is only the redundant part.

**Comment by SylvainCorlay** [comment]
Good catch @kiwi0fruit ! This should be fixed now!

**Comment by bollwyvl** [comment]
Looking good!

Once more for good luck: @conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,49 +1,56 @@
...
+    - tqdm >=4.42.1,<5.0.0a0
     - certifi
 
 test:
+  requires:
+    - pip
   imports:
     - pyppeteer
+  commands:
+    - python -m pip check
```
Comment (bollwyvl) [inline recipe/meta.yaml:41]: Shouldn't block this PR, but I really wish we had some kind of smoke test (if not the actual upstream tests) for whether it was going to work... there are a couple of examples around for the `yum` packages needed...

**Comment by bollwyvl** [comment]
LGTM, will let @kiwi0fruit punch it!

**Pull request merged**